Gemfile.lock

```diff
@@ -578,9 +581,8 @@ GEM
       http-cookie (>= 1.0.2, < 2.0)
       mime-types (>= 1.16, < 4.0)
       netrc (~> 0.8)
-    rexml (3.2.5)
-    rollbar (2.19.1)
-      multi_json
+    rexml (3.2.6)
+    rollbar (2.27.1)
```

Comment:
### [rollbar](https://rollbar.com/)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.19.1 → 2.27.1

[_[change-log](https://github.com/rollbar/rollbar-gem/releases), [source-code](https://github.com/rollbar/rollbar-gem), [gem-diff](https://my.diffend.io/gems/rollbar/2.19.1/2.27.1)_]





<details>
<summary>Commits</summary>

A change of **283** commits. See the full changes on [the compare page](https://github.com/rollbar/rollbar-gem/compare/v2.19.1...v2.27.1).

These are the first 10 commits:
- (4d2dda8) [841: use built-in rspec subject method](https://github.com/rollbar/rollbar-gem/commit/4d2dda84bd67a5ec8642997f72870165e397ecdd)
- (6ada0b4) [841: use delete instead of delete! as delete! returns nil if…](https://github.com/rollbar/rollbar-gem/commit/6ada0b4987b5a752ac0879c89768ae040027a23f)
- (2ebef5d) [841: remove proxy and dry_run form the request body](https://github.com/rollbar/rollbar-gem/commit/2ebef5d29f6584401b3419019a0b59d6d207a7af)
- (40f1b68) [841: loosen access to private methods and rename access_toke…](https://github.com/rollbar/rollbar-gem/commit/40f1b68b7a9291ef7bf5102f740a2321939b868d)
- (893efd6) [841: add support for unauthorized access token simulation](https://github.com/rollbar/rollbar-gem/commit/893efd6fcd4488ba9ce07279765380746e69f423)
- (1ef48cb) [841: don't pass json data to unauthorized - no need](https://github.com/rollbar/rollbar-gem/commit/1ef48cb947762ab90cf61f0efbab50c133a0a0a9)
- (485a2a9) [841: take under consideration dry run when determining succe…](https://github.com/rollbar/rollbar-gem/commit/485a2a975d7716753358db7b72f254af87916b3f)
- (f436edd) [841: add deploy spec and mock out the deploy api](https://github.com/rollbar/rollbar-gem/commit/f436edde280c6351cba6b8ac0f6e994fd99274c4)
- (66f643d) [841: loosen access to methods to match the parent](https://github.com/rollbar/rollbar-gem/commit/66f643d5821ff06cccaa6782236e732dbaaa56c9)
- (87b0569) [841: fix method name](https://github.com/rollbar/rollbar-gem/commit/87b0569e589b74e877f6ac021f79c1d25a086cdc)


</details>